observer/bin/spark-observer.js

```diff
@@ -47,6 +44,14 @@ const loop = async (name, fn, interval) => {
     }
     const dt = Date.now() - start
     console.log(`Loop "${name}" took ${dt}ms`)
+
+    recordTelemetry(`loop_${slug(name, '_')}`, point => {
+      point.intField('interval_ms', interval)
+      point.intField('duration_ms', dt)
+      point.intField('delay_ms', interval - dt)
```

Comment:
This field is redundant as it can easily be computed through the other ones
```suggestion
```

Reply:
In the past, I had bad experience with using computed fields in Grafana, but it was probably a different situation. I am fine to remove this field now, and bring it back only if I need it.